ts/test-unit/html/index_test.ts

```diff
@@ -0,0 +1,96 @@
+import 'mocha';
```

Comment:
I don't think we want a `ts/test-unit` directory. I think the right answer here is to put everything currently in `ts/test` into `ts/styleguide` and then this would end up in `ts/test`.